packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -207,7 +207,8 @@ export function applyFieldOverrides(options: ApplyFieldOverrideOptions): DataFra
 
       // Attach data links supplier
       newField.getLinks = getLinksSupplier(newFrame, newField, fieldScopedVars, context.replaceVariables, {
-        theme: options.theme,
+        // TODO needed?
```

Comment:
Yah, i think it's needed as it's passed down and used by display values proxy https://github.com/grafana/grafana/blob/bad048b7ba1fecd28e76509c545a67fc8b70fdeb/packages/grafana-data/src/field/getFieldDisplayValuesProxy.ts#L13